nemo_skills/evaluation/evaluator/koifeval.py

```
@@ -0,0 +1,1448 @@
+        text = text.replace("Ph.D.", "Ph<prd>D<prd>")
+
+    text = re.sub(
+        _MIXED_ALPHABETS + "[.]" + _MIXED_ALPHABETS + "[.]" + _MIXED_ALPHABETS + "[.]",
+        "\\1<prd>\\2<prd>\\3<prd>",
+        text,
+    )
+    text = re.sub(_MIXED_ALPHABETS + "[.]" + _MIXED_ALPHABETS + "[.]", "\\1<prd>\\2<prd>", text)
+
+    text = re.sub(" " + _SUFFIXES + "[.]", " \\1<prd>", text)
+    text = re.sub(" " + _ALPHABETS + "[.]", " \\1<prd>", text)
+    text = re.sub(r"\s" + _ALPHABETS + "[.]\s+(?=[가-힣])", " \\1<prd> ", text)
```

Ruff (0.15.10)

[warning] Invalid escape sequence: `\s`

Use a raw string literal

(W605)